### ISSUE TITLE
RFE: support to replace gnome-keyring

### ISSUE THREAD

**Issue by gfa** [vodik/envoy#38]
hello, 
    do you plan to implement the dbus interfaces/whatever needs to make envoy a replacement for gnome-keyring? 
ideally i would use envoy as a secret storage.

thanks!

**Comment by vtolstov** [comment]
2015-03-09 4:15 GMT+03:00 gfa <notifications@github.com>:

> hello,
> do you plan to implement the dbus interfaces/whatever needs to make envoy
> a replacement for gnome-keyring?
> ideally i would use envoy as a secret storage.
>
> thanks!
>

Nice idea!


-- 
Vasiliy Tolstov,
e-mail: v.tolstov@selfip.ru
jabber: vase@selfip.ru

**Comment by vodik** [comment]
@gfa Yes, it is something I've been seriously considering. I'm waiting for kdbus/session dbus stuff to get worked out in systemd land first. Then I'll port all of envoy's IPC onto dbus as well.

**Comment by gfa** [comment]
cool, i will start the work to get envoy into debian

**Comment by bjmgeek** [comment]
There's a WNPP for it already:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=758651
